test/fixtures/repo-symlinks/dot-git/hooks/applypatch-msg.sample

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
How do you feel about dropping all the sample hooks and other non-essential files from the fixture repo, just to keep things tidy?